README.md

```diff
@@ -12,19 +12,98 @@ other components of the system, including the Istio mixer and the Istio proxy me
 If you have a question about the Istio Manager or have a problem using it, please
 [file an issue](https://github.com/istio/manager/issues/new).
 
-## Build instructions ##
+## Setting up a local dev environment ##
```

Comment:
By the way, Mac build is possible as long as you use a case-sensitive partition, and hack build rules to do cross-compilation. Both issues eventually will go away now that Kubernetes faces the exact same problems.